README.md

```diff
@@ -122,7 +122,7 @@ To run [roslint](http://wiki.ros.org/roslint), use the following command with [c
 
 To run [catkin lint](https://pypi.python.org/pypi/catkin_lint), use the following command with [catkin-tools](https://catkin-tools.readthedocs.org/).
 
-    catkin lint -W2 --rosdistro kinetic
+    catkin lint -W2 --rosdistro melodic

```

Comment:
Nit: If you change all of these commands from `melodic` to `$ROS_DISTRO` it will reduce maintenance